pkg/kubectl/cmd/label.go

```diff
@@ -293,8 +293,9 @@ func (o *LabelOptions) RunLabel(f cmdutil.Factory, cmd *cobra.Command) error {
 				return err
 			}
 
+			fmt.Printf("Listing labels for %s/%s:\n", info.Mapping.GroupVersionKind.Kind, info.Name)
 			for k, v := range accessor.GetLabels() {
-				fmt.Fprintf(o.out, "%s=%s\n", k, v)
+				fmt.Fprintf(o.out, "  %s=%s\n", k, v)
```

Comment:
@juanvallejo Can you please paste an example output for `before` and `after`?

Reply:
Sure, will update PR description as well

**Before**
```
$ kubectl label dc myapp test-deployment-config label1=test --list
app=myapp
label1=test
label1=test
```

**After**
```
$ kubectl label dc myapp test-deployment-config label1=test --list
Listing labels for DeploymentConfig/myapp:
  label1=test
  app=myapp
Listing labels for DeploymentConfig/test-deployment-config:
  label1=test
```